[Fix]シングルトンになっているコンポーネントの受け取り方を変更、GearInfoのリストを他から加えられるようにした。

kuromaaaaa opened the pull request:
シングルトンになっているコンポーネントの受け取り方を変更、GearInfoのリストを他から加えられるようにした。
ScoreManagerが見せている数字に説明を付けた。

kuromaaaaa committed:
[Fix]シングルトンになっているコンポーネントの受け取り方を変更、GearInfoのリストを他から加えられるようにした。
ScoreManagerが見せている数字に説明を付けた。

ShiomiKazuma reviewed (commented):
処理は、大丈夫です。
UTF-8形式に直してください

kuromaaaaa committed:
[Fix]文字コードをUTF-8に変更

kuromaaaaa commented:
UTF-8に上書きしてプッシュしました

kuromaaaaa committed:
[add,fix]GameManagerの追加,GearManagerとScoreManagerの修正
[GameManager]ゲーム画面上のスコアの表示と更新、施設情報の表示と更新、カーソルがアップグレードされているかを管理するGameManagerを追加
[ScoreManager]スコアの表示が毎フレーム行われないようにスコアが増えた時だけ更新するようにGameManagerの関数を呼ぶようにした
[GearManager]GameManagerで施設情報を表示するときにソートされていてほしいのでソートする関数を作った

kuromaaaaa committed:
[fix]クリックのアップグレードとギアのアップグレードを分けた
こうすることでクリックだけ2倍、カーソルだけ２倍やババア２倍などにも対応できるようにした

ShiomiKazuma reviewed (commented):
修正ありがとうございます。
問題ないです。
